pkg/pods/pod.go

```diff
@@ -593,6 +596,34 @@ func (pod *Pod) Launchables(manifest Manifest) ([]launch.Launchable, error) {
 	return launchables, nil
 }
 
+func (pod *Pod) SetFinishExec(finishExec []string) {
+	var runAs string
+
+	if manifest, err := pod.CurrentManifest(); err == nil {
+		runAs = manifest.RunAsUser()
```

Comment:
sure, i guess it would have to be the pod's responsibility to drop privileges in the finish script just like the other launchables need to